### ISSUE TITLE
Add to packagist

### ISSUE THREAD

**Issue by razbakov** [EcomDev/EcomDev_PHPUnit#269]
Please submit your package to https://packagist.org/packages/submit

To make this command executable:
`composer require ecomdev/ecomdev_phpunit 0.3.7`

**Comment by robbieaverill** [comment]
It's not that easy with Magento 1.x - composer can't install Magento modules without needing to hack the core autoloader a bit

**Comment by razbakov** [comment]
@robbieaverill that is not true. magento modules installed perfectly without patching. but it is true about composer libraries. and to add this module to packagist is nothing to do with installation. it is just a list of repos for composer
